Frontend/ui-driver/src/Flow.purs

```diff
@@ -2984,5 +3016,24 @@ welcomeScreenFlow = do
   case welcomeScreen of
     GoToMobileNumberScreen -> loginFlow
 
+driverReferralScreenFlow :: FlowBT String Unit
+driverReferralScreenFlow = do
+  let (GlobalState defGlobalState) = defaultGlobalState
+  (GlobalState allState) <- getState
+  appConfig <- getAppConfigFlowBT Constants.appConfig
+  (ReferredDriversResp referredDriversResp) <- Remote.referredDriversBT (ReferredDriversReq "")
```

Comment:
It will be better if it is added in global events as a separate flow. and show shimmer untill you  get data from api.